src/Playlist.java

```diff
@@ -0,0 +1,13 @@
+/**
+ * Created by darionmoore on 11/28/16.
+ */
+public class Playlist {
+    int songGenre = [34];
```

Comment:
syntax error.  Remember you define an array with `int[] songGenres = new int[34]`.  Also if it is a list of genres then the variable should be plural